src/System.Diagnostics.FileVersionInfo/pkg/System.Diagnostics.FileVersionInfo.pkgproj

```diff
@@ -3,13 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Diagnostics.FileVersionInfo.csproj">
-      <SupportedFramework>net46;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
I'll clean these up to be consistent with the split packages by adding...

    <NotSupportedOnTargetFramework Include="netcore50">
      <PackageTargetRuntime>win7</PackageTargetRuntime>
    </NotSupportedOnTargetFramework>